### PR TITLE
live update both doggos and its sidecar

### PR DESCRIPTION
**Pull request by maiamcc**
Hello @jazzdan, @nicks,

Please review the following commits I made in branch maiamcc/live-update-multi-c:

434e44b25c88a404dee468411327dc635c1a9736 (2019-08-09 09:56:56 -0400)
live update both doggos and its sidecar

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: Tiltfile [hunk 1]
```diff
@@ -108,6 +108,20 @@ docker_build('spoonerisms', 'spoonerisms',
...
+docker_build('doggos', 'doggos',
+  live_update=[
+    sync('doggos', '/go/src/github.com/windmilleng/servantes/doggos'),
+    run('go install github.com/windmilleng/servantes/doggos'),
+    restart_container(),
+])
+docker_build('sidecar', 'sidecar',
+  live_update=[
+    sync('sidecar/src/', '/src/'),
+    run('cargo build --release'),
```
Comment (jazzdan) [inline Tiltfile]: Doesn't really matter but I bet IRL you'd just do a `cargo build` here since it would be faster and keep debug symbols.

Comment (maiamcc) [reply Tiltfile]: ha idk how to rust. I'll do that!

Comment (maiamcc) [reply Tiltfile]: ...wait nvm, it doesn't update properly if i do that 🤔

Comment (maiamcc) [reply Tiltfile]: curious why the thing you said doesn't work, but don't particularly feel like digging into it now, so imma merge the thing that works.

**Pull request merged**